adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -98,14 +98,14 @@ public DefaultDeviceResolver(
             final Tracer tracer,
             final String adapterName,
             final CoapAdapterProperties config,
-            final CredentialsClientFactory credentialsClientFactory,
+            final CredentialsClient credentialsClientFactory,
```

Comment:
The JavaDoc still contains information about the credentials client factory and needs update.
How about renaming the variable `credentialsClientFactory` to `credentialsClient`?